### ISSUE TITLE
Rodrom Development

### ISSUE THREAD

**Issue by timdog364** [hossumquat/Rodrom#7]
Hi Hossumquat,
For years I was following the development of MXP, the project that Rodrom is based upon. That project died and I'm wondering, what's happening with MXP? 
I am interested in reviving MXP but I don't think it's open source. Is it possible to get the MXP project files? 
Perhaps not, and maybe Rodrom would be a better place to start the revival of this great relic game. 
I had a message with Jason via the Dejenol forum about Rodrom, but that was years ago. I wonder if that account is Ghezra?

Any answers you can provide would be helpful, thanks!

**Comment by hossumquat** [comment]
I am Jason, and this is my project so I'm pretty certain that was me.  The code for MXP isn't open source and can't be due to legal agreements with the rights holder of the original Mordor, as I understand it. So I started this project from scratch.  I'd be happy to answer any questions you might have, just let me know.

**Comment by timdog364** [comment]
Hi Jason,
Do you still work on this project? I believe you were working on a game engine that could be used to build games in the image of Mordor. Is that engine fully developed?
Thanks,

**Comment by hossumquat** [comment]
Not finished, no.  I haven't been working on it recently but I do plan to
work on it again at some point.  You can grab the source and try getting it
running.  I used eclipse for my IDE it works for me but getting it up and
running in a more general context isn't really an area I have much
experience with yet.  If you run into issues let me know and I can probably
help you out.

**Comment by timdog364** [comment]
I just installed eclipse for Java and I'm going to begin learning how this project of yours works. I have a lot to learn but I hired a tutor and I'm also going to read tutorials. 
I will have some questions for you but I'll do my best to keep them brief. I'd like to open a more private chat channel with you elsewhere if possible, would you be open to that? I use Telegram, Signal, FB Messenger, Discord and a few others. What do you say?

**Comment by timdog364** [comment]
Yeah I'd like to open a private chat channel with you because I will have some questions, but I will try to keep them brief. Would you be open to that? I use Telegram, Discord, Whatsapp, FB Messenger and Signal and can use whatever you prefer.

**Comment by hossumquat** [comment]
I have a discord account so we could try do that.

**Comment by timdog364** [comment]
Ok add me on Discord: 
Chozami#5368
